web/massastation/src/utils/massaConstants.ts

```diff
@@ -7,3 +7,6 @@ export const PLUGIN_STOP = 'stop';
 export const PLUGIN_UPDATE = 'update';
 
 export const isZipFile = /^(http)[^\s]*\.zip$/i;
+
+// List of websites that are allowed to be added to favorites
+export const FAVORITES_WEBSITES = ['flappy', 'psychedelic', 'flappynathana1'];
```

Comment:
Let's move that to an specific folder for const. I would say the same we have in the wallet. Like that we can have a kind of standard.

web-frontend/src/const

Reply:
Feel free to close here when it's done!

Reply:
By bad i told @maxime-carabina tu put it here !! :/

Reply:
> Let's move that to an specific folder for const. I would say the same we have in the wallet. Like that we can have a kind of standard.
> 
> web-frontend/src/const

Already present in `src/utils/` 😃